src/proper_fsm.erl

```diff
@@ -153,8 +153,11 @@
 
 -export([commands/1, commands/2, run_commands/2, run_commands/3,
 	 state_names/1]).
--export([command/1, precondition/2, next_state/3, postcondition/3]).
+-export([targeted_commands/1, targeted_commands/2]).
+-export([command/1, precondition/2, next_state/3, postcondition/3,
+         all_commands/1]).
```

Comment:
I think it's better that `all_commands/1` appears in the same `-export(..)` as `select_commands/2` below.
Similarly for all other functions that are PropEr-private, and an appropriate comment is placed above that export declaration that these functions are not part of the API.